### PR TITLE
net-im/qtox: use icon and .desktop files from qtox git

### PR DESCRIPTION
**Pull request by suhr**
Subj.

### CODE REVIEW

File: net-im/qtox/qtox-9999.ebuild [hunk 1]
```diff
@@ -50,6 +50,6 @@ src_configure() {
 
 src_install() {
 	dobin "${S}/qtox"
-	doicon -s scalable "${FILESDIR}"/tox.svg
```
Comment (zetok) [inline net-im/qtox/qtox-9999.ebuild:53]: If it's no longer used, you can also remove it from overlay

Comment (suhr) [inline net-im/qtox/qtox-9999.ebuild:53]: Done.

**Comment by zetok** [comment]
could you squash those commits?

**Comment by suhr** [comment]
Done.

**Pull request merged**